lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -124,6 +127,7 @@ def void(authorization, options = {})
         endpoint = options[:cancel_or_refund] ? 'cancelOrRefund' : 'cancel'
         add_reference(post, authorization, options)
         add_network_transaction_reference(post, options)
+        add_recurring_detail_reference(post, options)
```

Comment:
These endpoints also don't support this field, https://docs.adyen.com/api-explorer/Payment/68/post/cancel and https://docs.adyen.com/api-explorer/Payment/68/post/cancelOrRefund

Reply:
Done!